js/npm/etherscan/README.md

```diff
@@ -1,34 +1,38 @@
 # @parity/etherscan
 
-A thin, lightweight promise wrapper for the api.etherscan.io/apis service, exposing a common endpoint for use in JavaScript applications.
+A thin, lightweight promise-based wrapper for the [`api.etherscan.io` service](https://etherscan.io/apis), exposing a common endpoint for use in JavaScript applications.
 
 [https://github.com/ethcore/parity/tree/master/js/src/3rdparty/etherscan](https://github.com/ethcore/parity/tree/master/js/src/3rdparty/etherscan)
 
-## usage
-
-installation -
+## installation
 
 ```
 npm install --save @parity/etherscan
 ```
 
-Usage -
+## usage
+
+You will need to include [`babel-polyfill`](https://github.com/babel/babel/tree/master/packages/babel-polyfill) as well as [a `Promise` polyfill](https://github.com/stefanpenner/es6-promise#readme) and [a `fetch` polyfill](https://github.com/matthew-andrews/isomorphic-fetch) if your target platform doesn't support them.
+
+```js
+require('babel-polyfill');
+require('es6-promise').polyfill();
+require('isomorphic-fetch');
```

Comment:
For fetch - could... maybe... use the package.json browser field (all modern bundler support this), and for the node entry include `isometric-fetch` for the browser one `whatwg-fetch`.

(Together with `babel-runtime` it could get us to a "just works" point.)